spec/warren/handler/broadcast_spec.rb

```diff
@@ -5,6 +5,11 @@
 require 'warren/handler/broadcast'
 
 RSpec.describe Warren::Handler::Broadcast do
+  # rubocop:todo RSpec/ExpectInHook
+  # rubocop:todo RSpec/StubbedMock
+  # rubocop:todo RSpec/MessageSpies
+  # rubocop:todo RSpec/MultipleMemoizedHelpers
+  # rubocop:todo RSpec/VerifiedDoubles
```

Comment:
Is it worth to move this file to the .rubocop_todo list?, seems a lot of exception cases.

Reply:
Nope. The problem with the .rubocop_todo is it just gets ignored, and no one realises that there are issues to be fixed. I've even seen people add *new* issues to it, which should *never* happen. By inlining the issues you make them visible, and encourage anyone modifying the code to fix them up.

Other issues with .rubocop_todo are:
1) Cops are disabled at the file-level, so if you've made a problem once in a file, its easy to keep making it, thereby making the problem harder to fix in future.
2) Metrics cops (Such as line length) are even worse, as they essentially will only warn you if you write something that is literally worse than everything else in your code base. In extreme cases this can make the cop almost entirely useles.

Since beginning to move todos inline, I've noticed I've become more proactive at fixing problems.